public/app/plugins/datasource/graphite/func_editor.ts

```diff
@@ -0,0 +1,318 @@
+import angular from 'angular';
+import _ from 'lodash';
+import $ from 'jquery';
+import rst2html from 'rst2html';
+
+export function graphiteFuncEditor($compile, templateSrv, popoverSrv) {
+  var funcSpanTemplate = '<a ng-click="">{{func.def.name}}</a><span>(</span>';
```

Comment:
variables for templates can be made `const` instead of `var` since they'll never changes.